packages/flame/test/game/flame_game_test.dart

```diff
@@ -659,11 +659,27 @@ void main() {
         },
       );
     });
+
+    group('Render box attachment', () {
+      testWidgets('calls on attach', (tester) async {
+        await tester.runAsync(() async {
+          var hasAttached = false;
+          final game = _OnAttachGame(() => hasAttached = true);
+
+          await tester.pumpWidget(GameWidget(game: game));
+          await game.toBeLoaded();
```

Comment:
I don't know why I'm so confused about this PR, but do we really need `toBeLoaded`, can't we just await for `game.onLoadFuture` here (if it looked like it used to before this PR)?